integration/simulation/simulation.go

```diff
@@ -323,72 +325,66 @@ func (s *Simulation) deployPublicCallbacksTest() {
 func (s *Simulation) deployTenZen() {
 	testlog.Logger().Info("Deploying ZenBase contract")
 
-	wg := sync.WaitGroup{}
```

Comment:
This waitgroup only had the one go routine in it, got rid of the go routine for clarity.